src/main/java/ch/njol/skript/registrations/EventValues.java

```diff
@@ -351,7 +372,11 @@ public T get(E e) {
 			}
 		};
 	}
-	
+
+	public static boolean doesExactEventValueHaveTimeStates(Class<? extends Event> e, Class<?> c) {
+		return getExactEventValueGetter(e, c, -1) != null || getExactEventValueGetter(e, c, 1) != null;
```

Comment:
```suggestion
	public static boolean doesExactEventValueHaveTimeStates(Class<? extends Event> event, Class<?> c) {
		return getExactEventValueGetter(event, c, -1) != null || getExactEventValueGetter(event, c, 1) != null;
```

Reply:
Is done so in pull request https://github.com/SkriptLang/Skript/pull/4858